### PR TITLE
feat(v3proxy): add total field conditionally to response

### PR DESCRIPTION
**Pull request by kschelonka**
As described

Also rearranged the graphql files since I'm going to be adding a lot more graphql queries/mutations/fragments.

### CODE REVIEW

File: servers/v3-proxy-api/src/graphql/fragments/ItemComplete.graphql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+fragment ItemComplete on Item {
```
Comment (kschelonka) [inline servers/v3-proxy-api/src/graphql/fragments/ItemComplete.graphql:1]: all these graphql files are just moved around/split into separate files. no changes except adding totalCount

File: servers/v3-proxy-api/src/graphql/queries/savedItemsByOffsetComplete.graphql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+query getSavedItemsByOffsetComplete($pagination: OffsetPaginationInput, $filters: SavedItemsFilter, $sort: SavedItemsSort) {
+    user {    
+        savedItemsByOffset(pagination: $pagination, filter: $filters sort: $sort) {
+            entries {
+                ...SavedItemComplete
+            }
+            totalCount
```
Comment (kschelonka) [inline servers/v3-proxy-api/src/graphql/queries/savedItemsByOffsetComplete.graphql:7]: this is the only changed line

File: servers/v3-proxy-api/src/graphql/queries/savedItemsByOffsetSimple.graphql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+query getSavedItemsByOffsetSimple($pagination: OffsetPaginationInput, $filters: SavedItemsFilter, $sort: SavedItemsSort) {
+    user {    
+        savedItemsByOffset(pagination: $pagination, filter: $filters sort: $sort) {
+            entries {
+                ...SavedItemSimple
+            }
+            totalCount
```
Comment (kschelonka) [inline servers/v3-proxy-api/src/graphql/queries/savedItemsByOffsetSimple.graphql:7]: this is the only changed line

**Review by bassrock — APPROVED** [review]
yay!

**Pull request merged**